spec/jobs/searchgov_domain_destroyer_job_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe SearchgovDomainDestroyerJob do
+  subject(:perform) { SearchgovDomainDestroyerJob.perform_now(args) }
+  let(:domain) { 'www.archive.gov' }
+  let!(:searchgov_domain) {
+    SearchgovDomain.create!(domain: domain, status: 200)
+  }
+  let(:args) {
+    { searchgov_domain: searchgov_domain }
+  }
+
+  describe '#perform' do
+    it 'requires a searchgov_domain as an argument' do
+      expect{SearchgovDomainDestroyerJob.perform_now}.
```

Comment:
stylistic nit: be sure to use spaces within curly brackets: 
`expect{ SearchgovDomainDestroyerJob.perform_now }`
https://github.com/rubocop-hq/ruby-style-guide#spaces-braces